README.md

```diff
@@ -394,10 +396,57 @@ The `extract` method takes no arguments:
     1. `v` must have the same type that `f` returns in `extend`.
 
 
+### Bifunctor
 
+A value that implements the Bifunctor specification is a functor with two
+type arguments that are both covariant.
```

Comment:
I have removed this sentence given there is no mention of type arguments elsewhere in the spec.